src/CommunityToolkit.Maui/Converters/ColorToColorConverters.shared.cs

```diff
@@ -8,7 +8,11 @@ namespace CommunityToolkit.Maui.Converters;
 public class ColorToBlackOrWhiteConverter : BaseConverterOneWay<Color, Color>
 {
 	/// <inheritdoc/>
-	public override Color ConvertFrom(Color value) => value.ToBlackOrWhite();
+	public override Color ConvertFrom(Color value)
+	{
+		ArgumentNullException.ThrowIfNull(value);
```

Comment:
If we made `ConvertFrom` protected would we still need to check if `value` is `null`? `Convert` in the base class is sealed so that behaviour cannot be overridden. I guess we cannot guarantee an implementation won't call it without passing `null` in for `value` though

This may rely on the newer behaviour added in #310 though.

Reply:
Yea, I'm curious how #310 will affect this as well. 

In general, a best-practice for C# libraries is to use `ArgumentNullException.ThrowIfNull` for parameters on all public APIs since users who don't have nullable enabled can + will still pass in `null`. (Side note - this is probably a good Proposal to implement, add to our contribution guidelines, and maybe implement an editorconfig rule if it's possible).

For all of the `ColorToColor` Converters, an extension method is immediately called, e.g. `value.ToBlackOrWhite()`. Since this would throw an `NullReferenceException` if a user passes in `null`, I figured a `ArgumentNullException` would be better + more-informative for the user.